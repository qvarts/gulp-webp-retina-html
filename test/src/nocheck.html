<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>No image existence check</title>
</head>
<body>
<img class="test-class" src="../img/no_test.png" alt="test" />
<img class="test-class" src="../img/no_test.jpg" alt="test" />
<img class="test-class" src="../img/no_test.gif" alt="test" />
</body>
</html>