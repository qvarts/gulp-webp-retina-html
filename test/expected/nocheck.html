<!doctype html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport"
          content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>No image existence check</title>
</head>
<body>
<picture>
<source srcset="../img/no_test.webp, ../img/no_test@2x.webp 2x" type="image/webp">
<source srcset="../img/no_test.png, ../img/no_test@2x.png 2x" type="image/png">
<img class="test-class" src="../img/no_test.png" alt="test" />
</picture>
<picture>
<source srcset="../img/no_test.webp, ../img/no_test@2x.webp 2x" type="image/webp">
<source srcset="../img/no_test.jpg, ../img/no_test@2x.jpg 2x" type="image/jpeg">
<img class="test-class" src="../img/no_test.jpg" alt="test" />
</picture>
<picture>
<source srcset="../img/no_test.webp, ../img/no_test@2x.webp 2x" type="image/webp">
<source srcset="../img/no_test.gif, ../img/no_test@2x.gif 2x" type="image/gif">
<img class="test-class" src="../img/no_test.gif" alt="test" />
</picture>
</body>
</html>